client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -87,6 +91,17 @@ const PluginsBrowserList = ( {
 		}
 	};
 
+	const spotlightOnClick = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_marketplace_spotlight_click', {
+				type: 'plugin',
+				slug: spotlightPlugin.slug,
+				id: spotlightPlugin.id,
+				site: site,
+			} )
+		);
+		page( `/plugins/${ spotlightPlugin.slug }/${ site }` );
```

Comment:
```suggestion
		page( `/plugins/${ spotlightPlugin.slug }/${ site || '' }` );
```
`site` can be null if there is no site selected:

![Screenshot_2022-03-01_14-49-42](https://user-images.githubusercontent.com/811776/156101346-5acc28e2-2b2a-4418-b361-5aba9602da3c.png)